cmd/start.go

```diff
@@ -87,25 +80,30 @@ func start(newProxy ProxyFactory) error {
 	return nil
 }
 
-func init() {
+func init() { //nolint:gochecknoinits
 	viper.SetEnvPrefix("TWINGATE")
 	viper.AutomaticEnv()
 
 	flags := startCmd.Flags()
-	flags.StringVar(&startFlags.CA, "ca", "../../test/data/ca.crt", "Root CA certificate")
-	flags.StringVar(&startFlags.TLSKey, "tls.key", "../../test/data/domain.key", "TLS key")
-	flags.StringVar(&startFlags.TLSCert, "tls.cert", "../../test/data/domain.crt", "TLS certificate")
-	flags.StringVar(&startFlags.K8sAPIServerToken, "k8sAPIToken", "", "k8s API Server Token")
-	flags.StringVar(&startFlags.Network, "network", "", "Twingate network ID. For example, network ID is autoco if your URL is autoco.twingate.com")
-	flags.StringVar(&startFlags.Host, "host", "twingate.com", "The Twingate service domain")
-	flags.BoolVarP(&startFlags.Debug, "debug", "d", viper.GetBool("DEBUG"), "Run in debug mode")
-
-	if err := viper.BindPFlag("network", flags.Lookup("network")); err != nil {
-		panic(fmt.Sprintf("failed to initialize: %v", err))
-	}
 
-	if err := viper.BindPFlag("host", flags.Lookup("host")); err != nil {
-		panic(fmt.Sprintf("failed to initialize: %v", err))
+	// Twingate flags
+	flags.String("network", "", "Twingate network ID. For example, network ID is autoco if your URL is autoco.twingate.com")
+	flags.String("host", "twingate.com", "The Twingate service domain")
+
+	// Gateway flags
+	flags.String("port", "8443", "Port to listen on")
+	flags.String("tlsCert", "", "Path to the TLS certificate for the Gateway")
+	flags.String("tlsKey", "", "Path to the TLS key for the Gateway")
+
+	// Kubernetes flags
+	flags.String("k8sAPIServerCA", "", "Path to the K8sAPIServerCA certificate for the Kubernetes API server")
+	flags.String("k8sAPIServerToken", "", "Bearer token to authenticate to the Kubernetes API server")
+
+	// Misc flags
+	flags.BoolP("debug", "d", false, "Run in debug mode")
+
+	if err := viper.BindPFlags(flags); err != nil {
+		panic(fmt.Sprintf("failed to bind flags: %v", err))
```

Comment:
- Rename `ca` flag to `k8sAPIServerCA`.
- Add `port` flag so it's configurable and passed to the proxy's config.
- Reorder flags into related groups.